conda_smithy/templates/github-actions.tmpl

```diff
@@ -26,11 +34,29 @@ jobs:
             UPLOAD_PACKAGES: {{ data.upload }}
         {%- if data.build_platform.startswith("osx") %}
             os: macos
+        {%- if github_actions.self_hosted %}
+            labels: ['macOS', 'self-hosted', 'x64']
+        {%- endif %}
```

Comment:
We probably can have M1 instances here, too?
```suggestion
        {%- if data.build_platform.startswith("osx-64") %}
            os: macos
        {%- if github_actions.self_hosted %}
            labels: ['macOS', 'self-hosted', 'x64']
        {%- endif %}
        {%- if data.build_platform.startswith("osx-arm64") %}
            os: macos
        {%- if github_actions.self_hosted %}
            labels: ['macOS', 'self-hosted', 'ARM64']
        {%- endif %}
```

Reply:
There's no support yet. See https://docs.github.com/en/actions/hosting-your-own-runners/about-self-hosted-runners#architectures

Reply:
Apparently it's possible to use Rosetta 2 to for the runner itself and then escape back to arm64: https://github.com/actions/runner/issues/805

Reply:
Great. We might need more changes. Can you open an issue to track?